taxi_service/settings.py

```diff
@@ -26,8 +28,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
The `DEBUG` setting is set to `True`. Remember to set it to `False` in a production environment to avoid exposing sensitive information.